README.md

```diff
@@ -107,46 +107,44 @@ These visualizations provide the most important data for club owners, and give t
+As you can see on the screenshot of our design, we have implemented a search bar to find specific reports. With the search bar, you can find reports for specific days and times, as well as a specific type of harassment.
 
 Next to the section with all the reports, you can see two pie charts. These charts visualize the amount of victims who asked for help or want to be in contact with FemAlert.
 
 ### 🔩 Settings
 
-On this page, the settings of the club owner are shown. The club owners can edit their information here, or turn on the Dark theme mode. Here's a screenshot of our design for the 'Settings' page:<br><br>
+On this page, the settings of the club owner are shown. The club owners can edit their information here, log out, or turn on the Dark theme mode. Here's a screenshot of our design for the 'Settings' page:<br><br>
 <img src="https://imgur.com/RzhchBy.png" width="500" height="300">
 
-As you can see, the information of the club and it's owner is being shown on this page. The club owner can also edit it's information, log out, or turn on the Dark theme mode.
-
-The Dark theme mode is an extra feature we are implementing if we have enough time to do so. Our main focus will be at the other pages, where the functionalities of the dashboard are the most important.
+The Dark theme mode is an extra feature we are implementing if we have enough time to do so. Our main focus will be on the other pages, where the functionalities of the dashboard are the most important.
 
 ## 👨🏼‍💻 Coding
 
 In this project, we are using Svelte and D3.js to create the dashboards.<br>
 Svelte is a JavaScript framework made to reduce boilerplate code. This makes your code more understandable and much easier to read.<br>
-D3.js is a JavaScript libary to visualize data and make various types of graphs, charts, and other visualizations.
+D3.js is a JavaScript library to visualize data and make various types of graphs, charts, and other visualizations.
 <br>
 
 Our working method in this project is based on an article written by Brad Frost. In this article, Brad Frost writes about Atomic Web Design. Atomic Web Design is a work method where you basically filter your application in multiple layers. <br><br>
-With Atomic Web Design, you are using Atoms, Molucules, Organisms, and Templates. Each group is a part of the next group. For example; multiple atoms form a molecule, multiple molecules form an organism, and so on. On the following picture you can see a visual representation of Atomic Web Design: <br>
+With Atomic Web Design, you are using Atoms, Molecules, Organisms, and Templates. Each group is a part of the next group. For example; multiple atoms form a molecule, multiple molecules form an organism, and so on. On the following picture, you can see a visual representation of Atomic Web Design: <br>
 
 <img src="https://imgur.com/zJcPlv4.png" width="500" height="300">
```

Comment:
I think this is the image of the atomic web design right? That image can be removed! Instead just briefly explain what it is 😸